packages/react/src/lib/getDisplayName.ts

```diff
@@ -0,0 +1,9 @@
+const getDisplayName = (Component: React.ReactType) => {
+  return (
+    (Component as React.ComponentType).displayName ||
+    (Component as React.ComponentType).name ||
+    (typeof Component === 'string' && Component.length > 0 ? Component : 'Unknown')
+  )
+}
+
+export default getDisplayName
```

Comment:
I will be there again with this link: https://github.com/gaearon/react-hot-loader#checking-element-types
We can't rely on `displayName`, so my proposal: let's move this util to `tests` directory to avoid accidental usages in component's code.

Reply:
sorry, just not sure why this won't work? yes, while types are not strictly equal (type vs proxied wrapper), there should be still the same set of React-specific static props preserved, such as `displayName`  (otherwise it won't be possible for React Debugger Tools to show display name). Don't get me wrong: I am fine to put this function to tests dir, but just don't see the problems with hot loader (have checked that locally) - am I missing something?

Reply:
> here should be still the same set of React-specific static props preserved
Yep, you're correct. 

https://github.com/Semantic-Org/Semantic-UI-React/pull/382#discussion_r75185603

May be this will be more correct case. I want to prevent a case when someone will rely on `
displayName` in production code.

See example: https://codesandbox.io/s/l77xq5y8nq

Reply:
lets prevent this - in either case, totally agree that we won't like this function to be used in prod code 👍